Jenkinsfile

```diff
@@ -47,6 +47,7 @@ for(i = 0; i < buildTypes.size(); i++) {
                             def mvnCmd = "mvn -Pdebug -U clean install javadoc:javadoc ${runTests ? '-Dmaven.test.failure.ignore=true' : '-DskipTests'} -V -B -Dmaven.repo.local=${pwd()}/.repository" 
                             if(isUnix()) {
                                 sh mvnCmd
+                                sh 'test `git status --short | tee /dev/stderr | wc --bytes` -eq 0'
```

Comment:
@jglick What is it the relation with [this](https://github.com/jenkinsci/jenkins/pull/2851#discussion_r113501480)?